src/sgame/sg_main.cpp

```diff
@@ -39,7 +39,7 @@ level_locals_t level;
 gentity_t          *g_entities;
 gclient_t          *g_clients;
 
-Cvar::Range<Cvar::Cvar<int>> g_showHelpOnConnection("g_showHelpOnConnection", "show MN_WELCOME on connect (1 = all, 2 = unregistered)", Cvar::NONE, 1, 0, 2);
+Cvar::Range<Cvar::Cvar<int>> g_showWelcome("g_showWelcome", "show MN_WELCOME on connect (0 = none, 1 = all, 2 = unregistered)", Cvar::NONE, 1, 0, 2);
```

Comment:
The default value should be 0

Reply:
Oops. Done.

Reply:
It's still 1.

Reply:
Done for real. Forgot to push from my main computer...